src/features/OrdersPage/components/OrdersPage.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {useEffect, useState} from 'react';
+import {SAMPLE_ORDERS} from '../../../constants';
+import Footer from '../../Footer/components/Footer';
+
+const OrdersPage = () => {
+  const [orders, setOrders] = useState([]);
+
+  useEffect(() => {
+    // currently setting orders manually. In future will fetch orders from API
+    setOrders(SAMPLE_ORDERS);
```

Comment:
why setting sample orders in non-dependent useEffect instead of direct initialisation in useState?

Reply:
done